rfcs/0006-runtime-references-interface.md

```diff
@@ -44,6 +45,11 @@ its builtin hash scanning and just use the file. Both files are
 newline-separated lists of store paths. Parse errors in the
 `runtime-references` file, or non-existent store paths, fail the build.
 
+To avoid duplicating the default reference scanning behavior in Nix,
+we could also make a tiny tool part of Nix itself that wraps the
+`scanForReferences` API, and then expose that in the `<nix>` namespace
+so tooling can get the full path to that tool.
```

Comment:
So should I leave something about it in, or remote it completely?

I'm thinking something like this:

```nix
stdenv.mkDerivation {
  name = "foo";
  src = "bar";
  scanner = import <nix/store-scanner.nix>;

  fixupPhase = ''
    $scanner $NIX_BUILD_TOP/build-time-requisites | grep -v foo > $NIX_BUILD_TOP/runtime-references
   '';
}
```

And then inside the